layer1/transaction/tests/setup_test.go

```diff
@@ -0,0 +1,40 @@
+//go:build integration
+
+package tests
+
+import (
+	"github.com/alicenet/alicenet/layer1/tests"
+	"github.com/alicenet/alicenet/logging"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+var HardHat *tests.Hardhat
+
+func TestMain(m *testing.M) {
+	hardhat, err := tests.StartHardHatNodeWithDefaultHost()
+	if err != nil {
+		panic(err)
+	}
+	HardHat = hardhat
+	code := m.Run()
```

Comment:
Add the hardhat.CLose() and m.Run() in a closure, so we guarantee (in theory) that we close the hardhat node even if the tests panics. Like this: https://github.com/alicenet/alicenet/blob/main/layer1/ethereum/tests/setup_test.go#L43
```suggestion
	func() {
		defer hardhat.Close()
		code = m.Run()
	}()
```